class-3-branches-reset/commands.txt

```diff
@@ -1,6 +1,9 @@
 git remote add <alias_name> <.git url>
 git remote rm <alias_name>
+git remote -v 
 git branch
 git checkout -b <branch_name>
 git checkout <branch_name>
-git switch <branch_name>
\ No newline at end of file
+git switch <branch_name>
+git reset <commit_id>  --hard/soft/mixed 
+git rebase <branch_you_want_to_copy> 
```

Comment:
@vikash123765 
please make a note that this command brings the changes from the branch mentioned in the command
to the branch u are on right note 
PS: copy seems ambiguous to me as a reader.